.spelling

```diff
@@ -46,6 +46,10 @@ ack-istio
 ACLs
 Acmeair
 ACS-Engine
+Add-ons
```

Comment:
We're been using the spelling 'addon' already (as seen in .spelling). I think we should be consistent one way or another.

Reply:
addon works for me

Reply:
I'm fine reverting to addon. Note that the istio.io/content/about/contribute/style-guide/index.md says to use add-on and not addon. We do seem to be inconsistent in the docs.

Reply:
Per Slack, @geeknoid Will fix the style guide to use addon.

Reply:
Changed to addon